app/components/footer.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from "@remix-run/react";
+import PrivacyPolicy from "~/routes/privacy_policy";
+
+import ReactDenverLogo from "../assets/react_denver_logo.svg";
+import YouTube from "../assets/YouTube.svg";
+import Discord from "../assets/Discord.svg";
+import GitHub from "../assets/GitHub.svg";
+import Twitter from "../assets/Twitter.svg";
+
+export default function Footer() {
+  return (
+    <div
+      style={{
+        height: "250px",
+        backgroundColor: "#A379C9",
+        display: "flex",
+        alignItems: "center",
+        paddingLeft: "5%",
+      }}
+    >
+      <img style={{ maxHeight: "50px" }} src={ReactDenverLogo} />
+      <a href="">
+        <img style={{ maxHeight: "50px" }} src={Discord} />
+      </a>
+      <a href="https://github.com/reactdenver">
+        <img style={{ maxHeight: "50px" }} src={GitHub} />
+      </a>
+      <a href="https://mobile.twitter.com/ReactDenver">
+        <img style={{ maxHeight: "50px" }} src={Twitter} />
+      </a>
+      <a href="https://www.youtube.com/c/ReactDenver">
+        <img style={{ maxHeight: "50px" }} src={YouTube} />
+      </a>
```

Comment:
Probably could use a css file and a className attribute. `<img className="brandingIcon" src={ReactDenverLogo} />`

Then a css file that you can import along with the other imports in the same file
`import './footerStyles.css'`

```css
.brandingIcon {
  maxHeight: 50px;
}
```

Reply:
Thanks for the feedback.  I wasn't sure how we wanted to structure the styling, whether it be in a separate file, inline, or in a style object like with React Native.  I just decided to put it inline for now.  From some searching it seemed that importing stylesheets with Remix wouldn't be as straightforward as with regular React, but maybe I misunderstood.